scripts/download.sh

```diff
@@ -47,12 +49,14 @@ if [ ! -f /var/spool/ctlstore/ldb.db ]; then
     pigz -d snapshot.db.gz
```

Comment:
Gently suggesting just one line of change here, not related to this ticket but it's too low a hanging fruit:
```suggestion
    pigz -df snapshot.db.gz
```
Why: [JIRA ticket](https://segment.atlassian.net/browse/CONFIG-333)

Reply:
wow, wild Roberto hit that edge case already. I'll change it and assign that ticket to me and this PR